llvm/test/tools/repo-fragments/tool-options-test.ll

```diff
@@ -0,0 +1,34 @@
+; Check repo-fragments options, which include:
+;    1) Check the default case by dumping _all_ of the names from a compilation;
+;    2) Check the -names option by dumping one or more fragments by names option;
```

Comment:
This sentence is confusing. I don’t think that you dump any fragments. How about “Check the ‑names option by explicitly requesting specific definitions by name”? I don’t think that you actually test the dumping of more than a single name.

Reply:
Changed it.